### PR TITLE
Fix : Workflow "start" paste url not support s3 pre-signed URL

### LINKED ISSUE #6718: Workflow "start" paste url not support s3 pre-signed URL
### Self Checks

- [X] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Dify version

0.6.14

### Cloud or Self Hosted

Self Hosted (Docker)

### Steps to reproduce

Create empty workflow "Start" -> "LLM" -> "End"

Start part, use paste image link, input a presigned URL will report error

### ✔️ Expected Behavior

can accept the pre-signed URL

### ❌ Actual Behavior

Report error "URL does not exist"
image

### PR DESCRIPTION
**Pull request by ybalbert001**
# Checklist:

> [!IMPORTANT]  
> Please review the checklist below before submitting your pull request.

- [x] Please open an issue before creating a PR or link to an existing issue
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods

# Description

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue. Close issue syntax: `Fixes #<issue number>`, see [documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) for more details.

Fixes https://github.com/langgenius/dify/issues/6718

## Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [ ] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# Testing Instructions

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Test A
- [x] Test B

### CODE REVIEW

**Comment by ybalbert001** [comment]
I had already run "poetry lock" to update poetry.lock file and also submitted. Why this Pytest still throw error?

**Comment by laipz8200** [comment]
We have a `sync-poetry` and `update-poetry` scripts under `dev/`, have a try.

**Comment by ybalbert001** [comment]
> We have a `sync-poetry` and `update-poetry` scripts under `dev/`, have a try.

Fixed

File: api/pyproject.toml [hunk 1]
```diff
@@ -108,7 +108,8 @@ authlib = "1.3.1"
 azure-identity = "1.16.1"
 azure-storage-blob = "12.13.0"
 beautifulsoup4 = "4.12.2"
-boto3 = "1.34.136"
+boto3 = "1.34.148"
+botocore = "^1.34.148"
```
Comment (bowenliang123) [inline api/pyproject.toml]: Avoid transparent dependencies for better alignment in versions,  since `botocore` is already required by `boto3`.

**Review by bowenliang123 — CHANGES_REQUESTED** [review]

File: api/pyproject.toml (hunk 1 above)
Comment (ybalbert001) [reply api/pyproject.toml]: Done

**Review by crazywoola — APPROVED** [review]

**Comment by laipz8200** [comment]
Great!

**Pull request merged**